share/smack/lib/smack.c

```diff
@@ -257,7 +257,13 @@ void* calloc(size_t num, size_t size) {
   return ret;
 }
 
+#ifdef __cplusplus
+
+void * operator new(size_t size) {
+	    return malloc(size);
```

Comment:
Wrong indentation here. I think you maybe used tab instead of spaces.

Reply:
Funny thing. This code should not be here - either I forgot to delete it or I botched something with git.